packages/sshnoports/lib/sshnp/sshnp_arg.dart

```diff
@@ -180,13 +185,27 @@ class SSHNPArg {
       format: ArgFormat.option,
       type: ArgType.string,
       allowed: SupportedSshClient.values.map((c) => c.cliArg).toList(),
+      commandLineOnly: true,
     ),
-    SSHNPArg(
+    const SSHNPArg(
       name: 'add-forwards-to-tunnel',
       defaultsTo: false,
       help: 'When true, any local forwarding directives provided in'
           '--local-ssh-options will be added to the initial tunnel ssh request',
       format: ArgFormat.flag,
+      commandLineOnly: true,
+    ),
+    const SSHNPArg(
+      name: 'config-file',
+      help: 'Read args from a config file\nMandatory args are not required if already supplied in the config file',
+      commandLineOnly: true,
+    ),
+    const SSHNPArg(
+      name: 'list-devices',
+      aliases: ['ls'],
+      negatable: false,
+      help: 'List available devices',
+      commandLineOnly: true,
```

Comment:
These were injected with an if, now using commandLineOnly: true to be injected to the command line